lib/failure.sh

```diff
@@ -245,6 +247,55 @@ fail_invalid_semver() {
   fi
 }
 
+log_other_failures() {
+  local log_file="$1"
+  if grep -qi "sh: 1: .*: not found" "$log_file"; then
+    mcount "failures.dev-dependency-tool-not-installed"
+  fi
+
+  if grep -qi "Failed at the bcrypt@\d.\d.\d install script" "$log_file"; then
+    mcount "failures.bcrypt-permissions-issue"
+  fi
+
+  if grep -qi "Versions of @angular/compiler-cli and typescript could not be determined" "$log_file"; then
+    mcount "failures.ng-cli-version-issue"
+  fi
+
+  if grep -qi "Versions of @angular/compiler-cli and typescript could not be determined" "$log_file"; then
```

Comment:
This check is a duplicate of the one above?

Reply:
Yeah, duplicate here!

Reply:
Nice catch @edmorley! 😅